### ISSUE TITLE
Back-end/gui cleanup

### ISSUE THREAD

**Issue by Nbrannon** [UCI-UAVForge/GroundStation#3]
A lot of no longer used functions and general useless junk has built up in the header and source files for the main windows, and preferably a returner from GUI or Back end needs to go and delete what isnt needed.

**Comment by aarcane** [comment]
A lot of replication in "back" or "Exit" style buttons, too.

**Comment by jason-watkins** [comment]
@Nbrannon, @mcraven, does #15 resolve this one?

**Comment by mcraven** [comment]
No, not until #21, #23, and #24 are complete at least.

**Comment by JordanD2** [comment]
Issue no longer relevant. More recent issues address these same and similar problems in better detail.
